lib/msal-browser/src/cache/IWindowStorage.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Interface object for window storage. TODO: Add link to custom cache docs.
```

Comment:
reminder for this link + docs

Reply:
Will work on docs once we have agreed on the implementation. Thanks!